### DISCUSSION TITLE
Buzzer Inav 4.0.0

### DISCUSSION THREAD

**Discussion by ghost** [iNavFlight/inav#7750] [Q&A] [answered]
Hello, I just flash from 3.0.1 to 4.0.0 and the most work well. Juste the Buzzer makes "problems". I remember in 3.0.0. then I install the Buzzer the motor or dshot buzzer was quit. Now in 4.0.0 both, the buzzer and motors makes sound. I install the buzzer so that the motors don't run hot while long buzzer time. So how I can silence the motors like in 3.0.0?
Thanks for your support.

**Comment by JulioCesarMatias** [comment, marked as answer]
> Hello, I just flash from 3.0.1 to 4.0.0 and the most work well. Juste the Buzzer makes "problems". I remember in 3.0.0. then I install the Buzzer the motor or dshot buzzer was quit. Now in 4.0.0 both, the buzzer and motors makes sound. I install the buzzer so that the motors don't run hot while long buzzer time. So how I can silence the motors like in 3.0.0? Thanks for your support.

CLI >> `set dshot_beeper_enabled = OFF` >> Save

**Comment by ghost** [comment]
Yes this works. But i check my dump file from 3.0.1 and there was the dshot_beeper still enabled
